content/06-vets-developer-docs/04-vets-website/03-forms/00-form-tutorial.md

```diff
@@ -19,27 +19,27 @@ yo @department-of-veterans-affairs/vets-website
 
 You can answer the questions with the [documentation](https://github.com/department-of-veterans-affairs/vets-website/blob/master/docs/GeneratorOptions.md) as a guide, or if you're just looking to try things out, here are the answers I'm using for this tutorial:
 
-```bash
-? What's the name of your application? This will be the default page title. My new form
-? What folder in src/js should your app live in? new-form
-? What should be the name of your app's entry bundle? newForm
-? What's the root url for this app? /new-form
-? Is this a form app? Yes
-? What's your form number? XX-230
-? What's the Google Analytics event prefix you want to use? new-form-
-? What's the respondent burden of this form in minutes? 30
-? What's the OMB control number for this form? XX3344
-? What's the OMB expiration date for this form? 5/31/2018
-? What's the benefit description for this form? new form benefits
-```
+- What's the name of your application? This will be the default page title. `My new form`
+- What folder in `src/applications/` should your app live in? This can be a subfolder. `new-form`
+- What should be the name of your app's entry bundle? `newForm`
+- What's the root url for this app? `/new-form`
+- Is this a form app? `Y`
+- What's your form number? `XX-230`
+- What's the Google Analytics event prefix you want to use? `new-form-`
+- What's the respondent burden of this form in minutes? `30`
+- What's the OMB control number for this form? `XX3344`
+- What's the OMB expiration date for this form? `5/31/2018`
+- What's the benefit description for this form? `new form benefits`
```

Comment:
Should we consider rephrasing this as "What type of benefits to which does this form relate?" Or "What type of benefits is this form for?" and then provide an example rather than the default value? e.g. health care benefits.
We should create a ticket for handling the addition of new forms for benefit types that currently only contain one form, as these would need to be reorganized similar to the education benefits forms.